characterization/characterization-project/build/tmp/jar/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
Add `build/` to the .gitignore
Use `git rm` to remove these temporary build files.
In fact, if this is just a separate FRC java project, you could just probably just copy the .gitignore from the main code to this project's directory.

Reply:
Nevermind, I got sick of scrolling past these files so I did it